app/_components/systems/Form.tsx

```diff
@@ -0,0 +1,334 @@
+import EconomiesField from '../inputs/Economies';
+
+export const SystemFormSchema = z.object({
+  system: z.string().regex(/[A-Za-z\ ]/),
+  onlyPopulated: z.enum(['1', '0']),
+  allegiance: z.enum(allegiances.map((item) => item) as [string, ...string[]]),
+  government: z.enum(governments.map((item) => item) as [string, ...string[]]),
+  primaryEconomy: z.enum(
+    economies.map((item) => item) as [string, ...string[]],
+  ),
+  minorFaction: z.string().regex(/[A-Za-z\ ]/),
+  presenceType: z.string().regex(/[A-Za-z\ ]/),
+  requiresPermit: z.enum(['1', '0']),
+  stationFilter: z.enum([
+    'hasStations',
+    'hasPlanetary',
+    'hasOrbital',
+    'hasNoStations',
+  ]),
+  powers: z.enum(powers.map((item) => item) as [string, ...string[]]),
+  powerEffects: z.enum(['Control', 'Expansion', 'Exploited']),
+  referenceSystem: z.string().regex(/[A-Za-z\ ]/),
+  securities: z.enum(securities.map((item) => item) as [string, ...string[]]),
+  factionStates: z.string().regex(/[A-Za-z\ ]/),
+});
+
+export type SubmitProps = z.infer<typeof SystemFormSchema>;
+
+interface FormProps {
+  onSubmitHandler: SubmitHandler<SubmitProps>;
+  isLoading: boolean;
+}
+
+const Form: React.FC<FormProps> = ({ onSubmitHandler, isLoading }) => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    control,
+  } = useForm<SubmitProps>({
+    resolver: zodResolver(SystemFormSchema),
+  });
+
+  const { isDark } = useColorMode();
+  const chakraSelectStyles = selectStyles();
+
+  const onSubmit: SubmitHandler<SubmitProps> = (data) => {
+    onSubmitHandler(data);
+  };
+
+  interface SelectGroup extends OptionBase {
+    label: string;
+    value: string;
+  }
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <Grid
+        templateColumns={{
+          base: 'repeat(1, 1fr)',
+          md: 'repeat(2, 1fr)',
+          lg: 'repeat(4, 1fr)',
+        }}
+        gap={6}
+        marginBottom="10"
+      >
+        <GridItem w="100%" colSpan={{ base: 1, md: 2, lg: 4 }}>
+          <FormControl
+            isInvalid={!!(errors.onlyPopulated && errors.onlyPopulated.message)}
+          >
+            <FormLabel>System</FormLabel>
+            <Input
+              variant="outline"
+              placeholder="Search by system name..."
+              borderColor={selectColor(isDark, 'border')}
+              _hover={{
+                borderColor: selectColor(isDark, 'border'),
+              }}
+              {...register('onlyPopulated', {
+                required: true,
+                pattern: /^[\w'-]+(?:\s[\w'-]+)*$/,
+                maxLength: 40,
+              })}
+            />
+            <FormErrorMessage>
+              {errors.onlyPopulated && errors.onlyPopulated.message}
+            </FormErrorMessage>
+          </FormControl>
+        </GridItem>
+
+        <GridItem w="100%" colSpan={{ base: 1, md: 2, lg: 4 }}>
+          <h5>
+            <b>SEARCH</b>
```

Comment:
Not sure about capitalisation for heirarchy personally, do we actually need this text here at all?